server/selective_sync_test.go

```diff
@@ -213,7 +213,7 @@ func TestChatMembersSpanPlatforms(t *testing.T) {
 	t.Run("user with empty id", func(t *testing.T) {
 		team := th.SetupTeam(t)
 		user1 := th.SetupRemoteUser(t, team)
-		chatMembersSpanPlatforms, appErr := th.p.ChatMembersSpanPlatforms(model.ChannelMembers{model.ChannelMember{UserId: user1.Id}, model.ChannelMember{UserId: ""}})
+		chatMembersSpanPlatforms, appErr := th.p.ChatMembersSpanPlatforms(model.ChannelMembers{model.ChannelMember{UserId: ""}, model.ChannelMember{UserId: user1.Id}})
```

Comment:
Is this particular change intentional?

Reply:
Actually, it was. `ChatMembersSpanPlatforms` now returns as soon as a remote user is found. So in order to catch this error, it has to happen on the first user. This function is called internally, so other than a test, a member would always have an ID. I didn't think it it necessary to loop all members just to check ID. I probably should have removed this test. 

This whole function will be removed with the PR that handles disconnected users.